src/target/stm32l4.c

```diff
@@ -680,6 +726,8 @@ bool stm32l4_probe(target_s *const target)
 			target->core = "M33+TZ";
 			break;
 		}
```

Comment:
This switch case is now missing an annotated fallthrough. A `break;` before `default:` should probably be added here.

Reply:
Ah, yep - good catch!